prow/cmd/prow-controller-manager/main.go

```diff
@@ -205,7 +205,7 @@ func main() {
 	}
 
 	if enabledControllersSet.Has(scheduler.ControllerName) {
-		if err := scheduler.Add(mgr, cfg, 1); err != nil {
+		if err := scheduler.Add(mgr, knownClusters, cfg, 1); err != nil {
```

Comment:
Since the scheduler needs a list of a sorted known clusters, then let's pass it that in this level instead of sorting them in Get

Reply:
It's not the scheduler but a particular strategy that needs to sort them out. Another one might just be fine with an unsorted collection. To be honest I'm not really opinionated whether `round-robin` should start with a sorted collection or not.

Reply:
I would agree that passing an unsorted list won't make any difference in the round robin operations.